{"uuid": "fb118882-5277-4d8a-ba6c-5edaa91b400d", "children": ["31c6732c-9d7f-470d-afe1-ae3e2c556b91"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "AttributeError: module 'selenium.webdriver' has no attribute 'webdriver'\n", "trace": "  File \"C:\\Users\\Office\\Pytest_framework\\pythonProject\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\Office\\Pytest_framework\\pythonProject\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Office\\Pytest_framework\\pythonProject\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Office\\Pytest_framework\\pythonProject\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Office\\Pytest_framework\\pythonProject\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Office\\Pytest_framework\\pythonProject\\conftest.py\", line 6, in setup\n    request.cls.driver= webdriver.webdriver()\n                        ^^^^^^^^^^^^^^^^^^^\n"}, "start": 1730114728314, "stop": 1730114728314}], "start": 1730114728314, "stop": 1730114728545}